main.go

```diff
@@ -97,6 +101,41 @@ func parseVariables(cmd *cobra.Command, args []string) {
 		}
 	}
 
+	templateJSONVariables = make(map[string]map[string]interface{})
+	for _, v := range jsons {
+		var (
+			jsonBytes []byte
+			err       error
+		)
+		key, jsonFileName, err := splitVarParam(v)
+		if err != nil {
+			fmt.Fprint(cmd.OutOrStderr(), err)
+			os.Exit(1)
+		}
+		if jsonFileName == "-" {
```

Comment:
*   Make sure to only have one value using "-"
*   Check for double keys, error out